bin/Powheg/runcmsgrid_powheg.sh

```diff
@@ -373,8 +373,8 @@ if [ -s pwg-stat.dat ]; then
   tail -${tail} cmsgrid_final.lhe                           >  cmsgrid_final.lhe_tail
   head -${head} cmsgrid_final.lhe                           >  cmsgrid_final.lhe_F
   proclin=`expr $head + 1`
-  PROCESS=`sed -n -e ${proclin}p  cmsgrid_final.lhe |  awk '{print $4}'`
-  echo "  "$XSECTION"   "$XSECUNC"  1.00000000000E-00 "$PROCESS >>  cmsgrid_final.lhe_F
+  proc=`sed -n -e ${proclin}p  cmsgrid_final.lhe |  awk '{print $4}'`
```

Comment:
@perrozzi : If you look at the beginning of the script, there's a tag with the same name. It will be replaced with the powheg process name by run_pwg.py. As "ggH_quark-mass-effect" has '-' in the process name, it will be a problem. But actually, this variable will be an empty string which does nothing after all, this won't affect the result even if not fixed.

Just that, I'm not sure if this evaluation yields an empty string is correct or not.
PROCESS=`sed -n -e ${proclin}p  cmsgrid_final.lhe |  awk '{print $4}'`